README.md

```diff
@@ -159,6 +159,39 @@ yarn build:integration
 yarn test:integration
 ```
 
+## Branching Model and Releases
+
+<!-- TODO: explain about changesets + how we do npm publishing + docker publishing -->
+
+### Active Branches
+
+| Branch          | Status                                                                           |
+| --------------- | -------------------------------------------------------------------------------- |
+| [master](https://github.com/ethereum-optimism/optimism/tree/master/)                   | Accepts PRs ONLY from `develop`                                                  |
+| [develop](https://github.com/ethereum-optimism/optimism/tree/develop/)                 | Accepts PRs that are compatible with `master` OR from `regenesis-X.X.X` branches |
+| [regenesis-0.4.0](https://github.com/ethereum-optimism/optimism/tree/regenesis-0.4.0/) | Deployed to Goerli + Kovan. **Branch frozen**. Merge to `develop` pending tests. |
+| [regenesis-0.5.0](https://github.com/ethereum-optimism/optimism/tree/regenesis-0.5.0/) | Active branch for next set of non-backwards-compatible changes.                  |
```

Comment:
Can we generalise `regenesis` branch description. Otherwise this will go out of date as soon as we merge `regenesis-0.4.0` or we start of a `regenesis-0.6.0` branch